BMM.Core/Helpers/GlobalConstants.cs

```diff
@@ -59,5 +59,7 @@ public class GlobalConstants
 
         public const string AndroidUpdateLink = "https://play.google.com/store/apps/details?id=org.brunstad.bmm";
         public const string IosUpdateLink = "itms-apps://apps.apple.com/no/app/bmm-brunstad/id777577855";
+
+        public const string UserVoiceLink = "https://uservoice.bcc.no/?tags=bmm";
```

Comment:
can you move this to FirebaseRemoteConfig? I already added it in firebase under "user_voice_link"